src/app/docs/inline-form/inline-form-docs.component.ts

```diff
@@ -1,11 +1,64 @@
 import {
-  ChangeDetectionStrategy,
+  ChangeDetectorRef,
   Component
 } from '@angular/core';
 
+import {
+  SkyDocsDemoControlPanelChange,
+  SkyDocsDemoControlPanelRadioChoice
+} from '@skyux/docs-tools';
+
+import {
+  SkyInlineFormButtonLayout,
+  SkyInlineFormCloseArgs,
+  SkyInlineFormConfig
+} from '../../public/public_api';
+
 @Component({
   selector: 'app-inline-form-docs',
-  templateUrl: './inline-form-docs.component.html',
-  changeDetection: ChangeDetectionStrategy.OnPush
+  templateUrl: './inline-form-docs.component.html'
 })
-export class InlineFormDocsComponent { }
+export class InlineFormDocsComponent {
+
+  public inlineFormButtonLayoutChoices: SkyDocsDemoControlPanelRadioChoice[] = [
+    { label: 'DoneCancel', value: SkyInlineFormButtonLayout.DoneCancel },
```

Comment:
I can see where it's implemented, but the demo I see uses Done and Cancel buttons. Is that a typical use case? It seems to me that given that there is an input field, Save and cancel buttons make more sense here and are probably the more typical use case.

Reply:
Good catch. Done.